src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -109,6 +109,9 @@ public override void OnActionExecuted(ActionExecutedContext context)
                     case OperationNotImplementedException _:
                         operationOutcomeResult.StatusCode = HttpStatusCode.NotImplemented;
                         break;
+                    case NotAcceptableException _:
```

Comment:
can you add a test to validate this? We have tests in `OperationOutcomeExceptionFilterTests.cs`

Reply:
Good idea - added in 5f34245.